### ISSUE TITLE
Toggle between ordered and unordered lists

### ISSUE THREAD

**Issue by Olisse** [Zettlr/Zettlr#5003]
### Checklist

- [X] Zettlr does not possess this feature (if it does, but the feature doesn't work out for you, propose an enhancement instead.)
- [X] Zettlr's current features cannot be used to approximate this behaviour (otherwise, propose an enhancement instead.)
- [X] I have searched the documentation to make sure I did not overlook it.
- [X] I have searched both open and closed issues to see if someone has already proposed this.

### Feature Proposal

In the past, I've used Atom with the [Markdown-Writer package](https://github.com/zhuochun/md-writer).
This package offers numerous commands and keyboard shortcuts. 
I really miss two shortcuts on Zettlr. For example : 
Shift-cmd-U for unordered list
Shift-cmd-O for ordered list

At first glance, this seems less practical than typing **+** or **1.** 
These seem superfluous but are actually very useful. These shortcuts can therefore be used as very fast toggles. 

### Example Workflow

First exemple. Often when we make a list, we start with a certain format but want to change. Let's say I write : 
`- first item `
If I place the cursor in this line and press Shift-cmd-O, it becomes
`1. first item`

If I select a complete list, this toggle also works. If I write : 
```
first item
second item
third item
```
And if I select all followed by the Shift-cmd-O command, it becomes
```
1. first item
2. second item
3. third item
```

More than this, if I place myself on the second line with the Shift-cmd-U shortcut, it becomes : 
```
1. first item
- second item
3. third item
```

Then, the list can then be easily completed with these new arguments by pressing the "enter" key: 
```
1. first item
- second item
- 
3. third item
4. 
```

### Caveats

I don't think this implementation conflicts with other functions offered by Zettlr. 

### Do you wish to attempt implementing this yourself?

No

### Platforms

- [ ] Windows
- [X] macOS
- [ ] Linux

**Comment by Olisse** [comment]
Let me complete my argument. 
If you don't have keyboard shortcuts, you can use the context menus to insert lists. But some strange behaviour prevents you from making the switch. 
For example : 

I create an ordered list: 
`1. First item`

Then, using the context menu, I ask to insert an unordered list. Logically, because there is no toggle, this becomes : 
`* 1. first item`

If you press Enter, it continues the list with the ordered and not the unordered (with a small indent)

`* 1. first item`
`  2. `

In conclusion, when you want to change from one format to another, you have to change everything by hand, with a lot of characters to delete, and it takes a long time. Especially when you're taking quick notes at meetings.
